package.json

```diff
@@ -12,7 +12,7 @@
   },
   "scripts": {
     "install-all": "npm-run-posix-or-windows install-all:task",
-    "install-all:task": "yarn & yarn install-launcher & yarn install-cli & yarn install-extension & yarn install-viewer & wait",
+    "install-all:task": "yarn; yarn install-launcher & yarn install-cli & yarn install-extension & yarn install-viewer & wait",
```

Comment:
sidenote while you're in here: can we move `yarn;` to happen in install-all while we're at it so it doesn't fail with "npm-run-posix-or-windows not installed" :)

Reply:
good point :)

Reply:
yah that just means we gotta do those two sequentially...

     "install-all": "yarn; npm-run-posix-or-windows install-all:task",